tap_bing_ads/__init__.py

```diff
@@ -688,9 +686,7 @@ def stream_report(stream_name, report_name, url, report_time):
         response = SESSION.get(url, headers={'User-Agent': get_user_agent()})
 
     if response.status_code != 200:
-        raise Exception('Non-200 ({}) response downloading report: {}'.format(
-            response.status_code, report_name))
-
+        raise Exception('Non-200 ({}) response downloading report: {}'.format(response.status_code, report_name))
```

Comment:
Reverte this change if it is not throwing any pylint error.

Reply:
done